### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Added navbar and app logo
- Added some styles to navbar
- Added new styles for links
- Added final line in the SCSS files

## Screenshot
Captura de Pantalla 2021-05-12 a la(s) 10 57 14 a  m

## Trello Card
https://trello.com/c/nVVbkQyG/31-navbar

### CODE REVIEW

File: books/scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.book-navbar {
```
Comment (mcallegari10) [inline books/scss/navbar.scss]: I think it's not a book navbar, but a navbar 😝

File: books/list.html [hunk 2]
```diff
@@ -9,6 +9,13 @@
         <link rel="stylesheet"  type="text/css" href="application.css">
     </head>
     <body>
+        <div class="book-navbar">
+            <div class="book-navbar-line"></div>
+            <div class="book-navbar-container">
```
Comment (mcallegari10) [inline books/list.html]: This should be a `nav` tag

File: books/list.html [hunk 3]
```diff
@@ -9,6 +9,13 @@
         <link rel="stylesheet"  type="text/css" href="application.css">
     </head>
     <body>
+        <div class="book-navbar">
+            <div class="book-navbar-line"></div>
```
Comment (mcallegari10) [inline books/list.html]: I think this is not needed. Try to solve this using CSS 😉

File: books/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  .book-navbar-container {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    max-width: 888px;
```
Comment (mcallegari10) [inline books/scss/navbar.scss]: I'm seeing this value repeated, maybe you can move this to a variable?

File: books/scss/styles.scss [hunk 5]
```diff
@@ -41,3 +41,8 @@ body {
   font-size: 20px;
   line-height: 24px;
 }
+
+.link-1 {
```
Comment (mcallegari10) [inline books/scss/styles.scss]: Adding numbers to the class names is not a really good practice. Try to avoid that please 😄

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: books/scss/styles.scss (hunk 5 above)
Comment (Julianhm9612) [reply books/scss/styles.scss]: Have been improved

File: books/scss/navbar.scss (hunk 4 above)
Comment (Julianhm9612) [reply books/scss/navbar.scss]: Have been improved

File: books/list.html (hunk 2 above)
Comment (Julianhm9612) [reply books/list.html:14]: Have been improved

File: books/scss/navbar.scss (hunk 1 above)
Comment (Julianhm9612) [reply books/scss/navbar.scss]: Names have been improved :sweat_smile:

File: books/list.html (hunk 2 above)
Comment (Julianhm9612) [reply books/list.html]: Has been changed to a nav element

File: books/list.html (hunk 3 above)
Comment (Julianhm9612) [reply books/list.html]: It was replaced by a border

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**